examples/migration_guide/bet_disk.py

```diff
@@ -0,0 +1,168 @@
+"""
+This module helps the users to use their old json files 
+containing BETDisks in the new version of Flow360.
+"""
+
+import json
+from typing import Optional
+
+import pydantic as pd
+
+from flow360.component.simulation.models.volume_models import BETDisk
+from flow360.component.simulation.primitives import Cylinder
+from flow360.component.simulation.unit_system import AngularVelocityType, LengthType, u
+
+
+def bet_disk_convert(
+    file: str,
+    save: Optional[bool] = pd.Field(False),
```

Comment:
interesting, is pd.Field() working here? this function is not pydantic model so I guess it is doing nothing here at most. Suggest:
```suggestion
    save: Optional[bool] = False,
```